src/utils/concatClassNames.ts

```diff
@@ -1,11 +1,9 @@
-type ClassName = string | string[] | undefined | false | null;
+type ClassName = string | undefined | false | null;
 
 export default function concatClassNames(...classNames: ClassName[]): string {
```

Comment:
I don't see a unit test for this. But the coverage report says this is 100% covered. I think we should still have a unit test and maybe look into why the coverage report says 100%. 

I can only assume it's being classed as 100% because it's being implicitly tested through the other components.

Reply:
Agree, lets have tests, it's probably not showing up if it's being called by other tests.  The coverage report might show whether it thinks is covered.

Reply:
Yes, it's not showing because it's covered by another test. I will add a test for it